README.md

```diff
@@ -1 +1,251 @@
-# empty-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```

Comment:
Please remove all the unused commits from the readme file before submitting them to your repository.